src/api/endPoints.js

```diff
@@ -0,0 +1 @@
+export const RESTAURANT_DATA_END_POINT = "restaurants";
```

Comment:
directly hardcode this is in the fetch function of restaurant, because you're going to use it for one time